source/usage-examples/find.txt

```diff
@@ -3,3 +3,54 @@ Find Multiple Documents
 =======================
 
 .. default-domain:: mongodb
+
+You can find multiple documents in a collection by using the ``Find()``
+method.
+
+The following example passes a query filter to the ``Find()`` method,
+which matches documents in the ``zips`` collection if the value of their
+``pop`` field is less than or equal to *500*:
```

Comment:
```suggestion
``pop`` field is less than or equal to 500:
```

Suggestion: I don't think we generally use italics for values.

Reply:
Nathan initially suggested this formatting on my UpdateOne PR (#5). I thought I would keep it across usage example pages for consistency.

Reply:
Sorry wasn't aware of this new convention. In that case, sounds good!